src/TestCase.php

```diff
@@ -263,13 +263,25 @@ public static function setUpBeforeClass(): void
         static::setUpBeforeClassForInteractsWithWebDriverOptions();
 
         if (! isset($_ENV['DUSK_DRIVER_URL'])) {
-            static::startChromeDriver(['port' => 9515]);
+            static::defineChromeDriver();
         }
 
         parent::setUpBeforeClass();
         static::startServing();
     }
 
+    /**
+     * Define the ChromeDriver.
+     *
+     * @return void
+     *
```

Comment:
```suggestion
     *
     * @api     
     *
     * @return void
     *
```

Marking this as an API method.